engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/text/AnyToTextNode.java

```diff
@@ -64,4 +61,13 @@ private Text doComplexAtom(Atom atom, InteropLibrary displays, InteropLibrary st
     res = Text.create(res, ")");
     return res;
   }
+
+  @CompilerDirectives.TruffleBoundary
+  private String showObject(Object child) throws UnsupportedMessageException {
+    if (child instanceof Boolean) {
+      return (boolean) child ? "True" : "False";
+    } else {
+      return strings.asString(displays.toDisplayString(child));
+    }
```

Comment:
Why this special case here?

Reply:
Because I can't change the way `InteropLibrary` treats booleans, but I want to, so I have to resort to manually detecting this.